run.cmd

```diff
@@ -17,12 +17,11 @@ if not defined VisualStudioVersion (
 set Platform=
 
 :: Restore the Tools directory
-call %~dp0init-tools.cmd
+call %~dp0init-tools.cmd -quiet
```

Comment:
I would prefer we don't add such an option to the cmd (or the sh which looks like you overlooked). What happens if you instead just redirect to a log file?